### ISSUE TITLE
Extending db

### ISSUE THREAD

**Issue by ZigaVukcevicDev** [howdyai/botkit-storage-mongo#19]
Hi, 

I am building a bot which would track records of time. E.g.: user calls it like this `/work start` and the record of date/time would be written.

I am looking for some ideas where to write this. Could I use the engine you wrote?

Thank you for pointers in advance.

Ziga

**Comment by benbrown** [comment]
@be-codified yes, you can use this to store information about the user, channel, or team in which your bot is present.

You can also now use the additional `tables` parameter to access other tables in your mongo collection. However, you will have to configure those models yourself, as this module does not provide that capability.
